website/docs/r/cs_kubernetes.html.markdown

```diff
@@ -96,6 +96,8 @@ The following arguments are supported:
 * `image_id` - Custom Image support. Must based on CentOS7 or AliyunLinux2.
 * `user_data` - (Optional, Available in 1.81.0+) Windows instances support batch and PowerShell scripts. If your script file is larger than 1 KB, we recommend that you upload the script to Object Storage Service (OSS) and pull it through the internal endpoint of your OSS bucket.
 * `exclude_autoscaler_nodes` - (Optional, Available in 1.88.0+) Exclude autoscaler nodes from `worker_nodes`. default: false 
+* `service_account_issuer` - (Optional, ForceNew) The issuer of the Service Account token for [Service Account Token Volume Projection](https://www.alibabacloud.com/help/doc-detail/160384.htm), corresponds to the `iss` field in the token payload. Set this to `"kubernetes.default.svc"` to enable the Token Volume Projection feature (requires specifying `api_audiences` as well).
```

Comment:
Please add a tag `Available in 1.89.0+` for new parameters.

Reply:
Done!